org.eclipse.lsp4e.jdt/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: JDT Integration for LSP4E
 Bundle-SymbolicName: org.eclipse.lsp4e.jdt;singleton:=true
-Bundle-Version: 0.13.5.qualifier
+Bundle-Version: 0.13.6.qualifier
```

Comment:
we do not need this for this PR, right?

Reply:
It appears so... The same sgoes for lsp4e plugin manifest and pom... without bumping up the version local build failed for me...

Reply:
Hmm... let me try reverting plugin version changes... wonder if the PR build would fail the same...